backtesting/lib.py

```diff
@@ -77,6 +78,23 @@ def barssince(condition: Sequence[bool], default=np.inf) -> int:
     return next(compress(range(len(condition)), reversed(condition)), default)
 
 
+def compute_stats(
+        trades: List[pd.DataFrame],
+        equity: np.ndarray,
+        ohlc_data: pd.DataFrame,
+        risk_free_rate: float = 0) -> pd.Series:
+    # TODO: Add details
+    """
+    Computes strategy performance metrics.
+
+    >>> broker = stats._strategy._broker
+    >>> equity = pd.Series(broker._equity).bfill().fillna(broker._cash).values
+    >>> perf = compute_stats(trades=broker.closed_trades, equity=equity, ohlc_data=GOOG)
```

Comment:
@kernc passing `stats._strategy._broker.closed_trades` makes it compatible as is. If we want to pass `stats._trades`, we'd need to convert the `DataFrame` to a list of `Trade`.